analyzer/optane/check_dimm_MediaTemperature_Injection_Counter

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+#set -x
+
+# Prevent repeated sourcing
+if [ -n "${_MODULE_CHECK_DIMM_MEDIA_TEMP_INJECTION_COUNTER_}" ]; then
+  return
+fi
+_MODULE_CHECK_DIMM_MEDIA_TEMP_INJECTION_COUNTER_="Loaded"
+
+# Validate Temp Injection Counter != 0
+function optane_check_dimm_media_temperature_injections_count() {
+  local FNAME=$1        # File name to process
+  local ERR_STATE=false # Used for error reporting
+
+  # Exit the function with INFO message if the file to process doesn't exist
+  if [[ ! -f ${FNAME} ]] ; then
+    info_msg " Could not process '${FNAME}'. File not found"
+    REPORT_COUNT_INFO=$((REPORT_COUNT_INFO+1)) # Increment the INFO count
+    return
+  fi
+
+  # Create Associative Array
+  declare -A MEDIA_TEMP_INJ_COUNETER  # Key is the DIMM Property value; Value is a list of DIMMs with this Property value
```

Comment:
Typo "COUNETER" should be "COUNTER". Please replace all occurrences in this file.